### ISSUE TITLE
Autoreactive cd21-/low cells

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#6231]
**Preferred term label:**

**Synonyms**

**Definition (free text, please give PubMed ID)**
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4479193/
https://pubmed.ncbi.nlm.nih.gov/29155103/

figure out term label and definition!

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
Comment: Complement receptor type 2 (CR2, CD21) is a 145‐kD glycosylated single polypeptide chain consisting of a long extracellular domain of 15 to 16 short consensus repeat sequences, a transmembrane region, and a short cytoplasmic domain. The receptor is expressed on B cells, follicular dendritic cells, thymocytes and a subset of peripheral T cells. CD21 binds complement fragments C3d, C3dg and iC3b that are covalently bound to target antigens. On B cells, CD21 forms a complex together with CD19 and CD81 (TAPA‐1), which functions as a co‐receptor to the BCR. Upon simultaneous binding of complement‐tagged antigens by CD21 and the BCR the threshold for B cell activation is reduced. Under these circumstances, CD21 acts as a bridge between the co‐receptor complex and the BCR. PMID:26119182

**Comment by pnrobinson** [comment]
Because CD21 augments B-cell receptor (BCR)-mediated signaling as part of the B-cell coreceptor complex, its down-regulation may confer a state of anergy to these cells, as has been demonstrated among CD21-/low B cells in patients with rheumatoid arthritis (RA), common variable immunodeficiency (CVID) or hepatitis C associated cryoglobulinemia patients. These CD21-/low B cells are enriched in autoreactive clones that are unresponsive to BCR stimulation, suggesting that these cells are controlled by the tolerizing mechanism of functional anergy.

**Comment by pnrobinson** [comment]
added
